test/parallel/test-buffer-safe-unsafe.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+const SlowBuffer = require('buffer').SlowBuffer;
+const assert = require('assert');
+
+const safe = Buffer.zalloc(10);
+const safeslow = SlowBuffer.zalloc(10);
+
+function isZeroFilled(buf) {
+  for (let n = 0; n < buf.length; n++)
+    if (buf[n] > 0) return false;
+  return true;
+}
```

Comment:
Yeah, I was going to get back to that to see. The best approach I can think of is to do a whole bunch of allocations and test all of them